kernel/src/main/java/org/kframework/compile/ResolveStrict.java

```diff
@@ -193,6 +159,55 @@ public K apply(KVariable var) {
                 sentences.add(ctx);
             }
         }
+    }
+
+    public Set<Sentence> resolve(Production production, boolean sequential) {
+        long arity = production.nonterminals().size();
+        List<Integer> strictnessPositions = new ArrayList<>();
+        List<Integer> allPositions = new ArrayList<>();
+        Set<ContextAlias> aliases = new HashSet<>();
+        String attribute;
+        Set<Sentence> sentences = new HashSet<>();
+        if (sequential) {
+            attribute = production.att().get(Att.SEQSTRICT());
+        } else {
+            attribute = production.att().get(Att.STRICT());
+        }
+        if (attribute.isEmpty()) {
+            for (int i = 1; i <= arity; i++) {
+                strictnessPositions.add(i);
+            }
+            aliases.add(DEFAULT_ALIAS);
+            resolve(sequential, sentences, arity, strictnessPositions, allPositions, aliases, production);
+            allPositions.addAll(strictnessPositions);
+        } else {
+            String[] components = attribute.split(";");
```

Comment:
Do we have a wrapper for 1-dimensional attribute arrays? I know it would just be `attribute.split(";")` still, but maybe it would be good to call that function if it exists for semantic readability.